app/controllers/EditorialSupportTeamsController.scala

```diff
@@ -17,6 +17,10 @@ object EditorialSupportTeamsController extends Controller with PanDomainAuthActi
     editorialSupportTable.putItem(EditorialSupportStaff(java.util.UUID.randomUUID().toString, name, false, team).toItem)
   }
 
+  def checkIfStaffExists(name: String, team: String): Boolean = {
+    if (findStaff(name, team).nonEmpty) true else false
```

Comment:
don't need to wrap this in an `if`, `findStaff(name, team).nonEmpty` would be ok